README.md

```diff
@@ -2,6 +2,7 @@
 Access design documentation for features slotted in the 4.3 release of OpenShift. Click links below for specific section details.
 
 - [Diagnostics](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/diagnostics/diagnostics)
+- [Set Default Storage Class](http://openshift.github.io/openshift-origin-design/web-console/Storage/set-default-storage-class/set-default-storage-class)
```

Comment:
Similar to the 4.2 section below, I think we should create an OCS subsection for this to live under. What do you think?

Reply:
Make sense, Liz. 
Currently, it is a link under 4.3 section and files are under 'storage' folder (outside 4.0 design folder) [refer the image attached]
Once we change everything to PF4, it should be moved to an appropriate folder structure. 

<img width="300" alt="image" src="https://user-images.githubusercontent.com/26006817/66694312-b9bbf880-eccf-11e9-8246-fa73d164b09c.png">

Reply:
@muruGanesan I like it living in this 4.3 section now. How about added an "OpenShift Container Storage" subsection similar to what we have today in the 4.2 area?